plugins/rocks.xml.webhelp/resources/js/xml.rocks.js

```diff
@@ -375,6 +375,7 @@ function closeSpan(className){
         if (openBreadcrumbs.classList.contains('show')) {
             openBreadcrumbs.classList.remove('show');
             $('.shading-container-wrapper').removeClass('grey');
+            $('.head-breadcrumbs-child-elements').children().removeClass('show');
```

Comment:
Is it working if we rotate tablet, not just expand size in the inspector?

Reply:
Yes, because these changes simply remove classes that were added in the showBreadcrumbs() function.